### PR TITLE
Update channels dependencies

### PR DESCRIPTION
**Pull request by sebastian-aranda**
This PR makes an upgrade on the Django channels dependencies in order to avoid possible errors related to the channels behavior.

### CODE REVIEW

File: manager/subscription/heartbeat_manager.py [hunk 1]
```diff
@@ -80,7 +84,7 @@ async def dispatch_heartbeats(cls):
             channel_layer = get_channel_layer()
             while True:
                 try:
-                    print("sending data")
+                    print("sending data", flush=True)
```
Comment (areyesd14) [inline manager/subscription/heartbeat_manager.py:87]: This print is for log reviews?

File: manager/subscription/heartbeat_manager.py [hunk 2]
```diff
@@ -58,14 +58,18 @@ async def query_commander(cls):
...
             while True:
                 try:
                     # query commander
                     resp = requests.get(heartbeat_url)
-                    timestamp = resp.json()['timestamp']
-                    #get timestamp
-                    cls.set_heartbeat_timestamp('Commander', timestamp)
+                    timestamp = resp.json()["timestamp"]
+                    # get timestamp
+                    cls.set_heartbeat_timestamp("Commander", timestamp)
```
Comment (areyesd14) [inline manager/subscription/heartbeat_manager.py:72]: Apparently single quotes make a difference jejejejeje

**Review by areyesd14 — APPROVED** [review]
Everything looks good. Great job!

I left you some comments but nothing relevant. We can discuss it at another time.

File: manager/subscription/routing.py [hunk 3]
```diff
@@ -4,6 +4,9 @@
 from .consumers import SubscriptionConsumer
 
 websocket_urlpatterns = [
-    url("^manager/ws/subscription/?$", TokenAuthMiddleware(SubscriptionConsumer)),
+    url(
+        "^manager/ws/subscription/?$",
+        TokenAuthMiddleware(SubscriptionConsumer.as_asgi()),
```
[the remark this answers is not in the document]
Comment (sebastian-aranda) [reply manager/subscription/routing.py:9]: Yes! I read this from here:
https://channels.readthedocs.io/en/stable/releases/3.0.0.html#update-to-asgi-3

File: manager/subscription/heartbeat_manager.py (hunk 1 above)
Comment (sebastian-aranda) [reply manager/subscription/heartbeat_manager.py:87]: Yes, to check when a heartbeat message is sent

Comment (sebastian-aranda) [reply manager/subscription/heartbeat_manager.py:87]: Yes, to check when a heartbeat message is sent

File: manager/subscription/heartbeat_manager.py (hunk 2 above)
Comment (sebastian-aranda) [reply manager/subscription/heartbeat_manager.py:72]: jeje it seems, this are automatic black formatting fixes

**Comment by sebastian-aranda** [comment]
This changes are being reviewed on branch `tickets/DM-28995`. After everything is verified a PR will be created from the mentioned branch.

**Pull request closed without merging**